src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,53 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { Sex } from '../types/Sex';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
```

Comment:
move all query param names to some enum it will help us to easily manipulate them